databricks-iris/{{ cookiecutter.repo_name }}/conf/base/catalog.yml

```diff
@@ -56,43 +56,43 @@ example_iris_data:
 # for all SparkDataSets.
```

Comment:
We could use this opportunity to update some of the links here? For example the `https://kedro.readthedocs.io` link above.

On the other hand, given that this already has a bunch of datasets defined, maybe it doesn't need as many code comments explaining how to do it?

Reply:
This is a good catch, thanks. I've updated the links here, though I think it would be good to check all starters for broken links also. I've opened an [issue](https://github.com/kedro-org/kedro-starters/issues/131) to tackle this.